uber/models.py

```diff
@@ -367,13 +376,14 @@ def __getattr__(self, name):
             return suffixed
 
         try:
-            [multi] = [col for col in self.__table__.columns if isinstance(col.type, MultiChoice)]
             choice = getattr(c, name)
-            assert choice in [val for val, desc in multi.type.choices]
+            if len(self.multichoice_columns) == 1:
+                multi = self.multichoice_columns[0]
+                if choice in multi.type.choices_dict:
+                    return choice in getattr(self, multi.name + '_ints')
         except:
```

Comment:
For some reason I can't reply directly to this comment:
> What's the functional difference between this and all_checkgroups? Is it that they return different types of things? I can see that all_checkgroups returns just the name -- I'm not super sure what it means to return a column instead.
>
> We might want to name these similar things, although given the choice I'd prefer this kind of naming scheme (describing what it is) rather than the 'checkgroups' name which describes more how it's used in an HTML form.

But yeah, the only difference is `multichoice_columns` returns the column objects. And it follows a fairly obvious naming scheme.